docs/kr.tree

```diff
@@ -197,6 +197,7 @@
             <toc-element id="mixing-java-kotlin-intellij.md"/>
             <toc-element id="jvm-records.md"/>
             <toc-element toc-title="Java to Kotlin migration guides">
+                <toc-element toc-title="Standard input" id="standard-input.md"/>
```

Comment:
I'd also place it at least after "Strings" or even make it the last in this section.